eng/pipelines/libraries/build-job.yml

```diff
@@ -93,56 +93,16 @@ jobs:
             displayName: Disk Usage after Build
 
         - ${{ if eq(parameters.runTests, false) }}:
-          - ${{ if ne(parameters.isOfficialBuild, true) }}:          
-            - task: CopyFiles@2
-              displayName: Prepare testhost folder to publish
-              inputs:
-                sourceFolder: $(Build.SourcesDirectory)/artifacts/bin/testhost
-                targetFolder: $(Build.ArtifactStagingDirectory)/artifacts/bin/testhost
-
-            - task: CopyFiles@2
-              displayName: Prepare runtime folder to publish
-              inputs:
-                sourceFolder: $(Build.SourcesDirectory)/artifacts/bin/runtime
-                targetFolder: $(Build.ArtifactStagingDirectory)/artifacts/bin/runtime
-
-            - task: CopyFiles@2
-              displayName: Prepare ref folder to publish
-              inputs:
-                sourceFolder: $(Build.SourcesDirectory)/artifacts/bin/ref
-                targetFolder: $(Build.ArtifactStagingDirectory)/artifacts/bin/ref
-
-          - task: CopyFiles@2
-            displayName: Prepare shared framework ref assemblies to publish
-            inputs:
-              sourceFolder: $(Build.SourcesDirectory)/artifacts/bin/ref/microsoft.netcore.app
-              targetFolder: $(Build.ArtifactStagingDirectory)/artifacts/bin/ref/microsoft.netcore.app
-
-          - task: CopyFiles@2
-            displayName: Prepare shared framework runtime folder to publish
-            inputs:
-              sourceFolder: $(Build.SourcesDirectory)/artifacts/bin/pkg
-              targetFolder: $(Build.ArtifactStagingDirectory)/artifacts/bin/pkg
-
+          - template: /eng/pipelines/libraries/prepare-publish-for-tests.yml
+            parameters:
+              isOfficialBuild: ${{ parameters.isOfficialBuild }}
+          
           - task: CopyFiles@2
```

Comment:
The docs should also be needed for the official build to produce packs with docs?

Reply:
We don't yet build the product build and the reason why I removed this is because I don't currently produce the docs folder with the build command so this fails unless I start producing them.

Reply:
The reason why I think we should keep `prepare-publish-for-tests` on pair with what we need on runtime and runtimelab is to avoid missing something or adding something to `prepare-publish-for-tests` that was in `build-job` because we didn't see it there. It is just weird to be in a `hybrid` mode of some things are in the shared template and some are not. You will eventually need to publish these, so why not put them there?

You could also put it there and disable via a parameter that the template takes.